### ISSUE TITLE
Getting ImportError even though the package is installed into python

### ISSUE THREAD

**Issue by srshearer** [Tautulli/Tautulli#1162]
**Version:**
1.4.25

**Branch:**
master

**Commit hash:**
cfd30996264b7e9fe4ef87f02d1cc52d1ae8bfca

**Operating system:**
macOS 10.12.6

**Python version:**
2.7.13 (default, Dec 20 2016, 15:49:03) [GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)]

**What you did?**
I set up a custom python script to send notifications to a slack channel. This script works properly when run by itself from Terminal but when it is triggered from plexPy I get an ImportError for any modules that I have installed. 

Beginning of Script:
```
#!/usr/local/bin/python -u
# encoding: utf-8

import os
import sys
import argparse
import json
import requests
```

**How can we reproduce your issue?**

**What are your (relevant) settings?**
```
$ pip show requests
Name: requests
Version: 2.18.4
Summary: Python HTTP for Humans.
Home-page: http://python-requests.org
Author: Kenneth Reitz
Author-email: me@kennethreitz.org
License: Apache 2.0
Location: /usr/local/lib/python2.7/site-packages
Requires: idna, urllib3, certifi, chardet
```

**Logs:**
```
PlexPy Notifiers :: Script error: 
    Traceback (most recent call last): 
        File "/Users/media/git/pyBots/slackBot/slackAnnounce.py", line 17, in 
            import requests 
    ImportError: No module named requests

2017-12-11 14:30:52 | ERROR | PlexPy Notifiers :: Script error:     Traceback (most recent call last):         File "/Users/media/git/pyBots/slackBot/slackAnnounce.py", line 17, in             import requests     ImportError: No module named requests
2017-12-11 14:30:52 | DEBUG | PlexPy Notifiers :: Executing script in a new thread.
2017-12-11 14:30:52 | DEBUG | PlexPy Notifiers :: Full script is: ['python', u'/Users/media/git/pyBots/slackBot/slackAnnounce.py', '-d', '-m', 'test']
2017-12-11 14:30:52 | DEBUG | PlexPy Notifiers :: Trying to run notify script, action: test, arguments: -d -m test
2017-12-11 14:30:52 | DEBUG | Sending test Scripts notification.
2017-12-11 14:30:52 | INFO | PlexPy Config :: Writing configuration to file
```

<img width="640" alt="1-scriptsettings" src="https://user-images.githubusercontent.com/4999779/33857569-148ba828-de81-11e7-93bb-8410afa63966.png">
<img width="601" alt="2-testscript" src="https://user-images.githubusercontent.com/4999779/33857570-14acf7a8-de81-11e7-8582-3a55f2b8d5dc.png">

**Comment by JonnyWong16** [comment]
Is it using the correct version of python? `python --version`

**Comment by srshearer** [comment]
It appears to be.
```
$ python --version
Python 2.7.13
```

PlexPy and the version of python that is run in Terminal both seem to be located at `/usr/local/bin/python` so I'm sort of at a loss.

**Comment by JonnyWong16** [comment]
I'm pretty sure you have two versions of python installed. PlexPy bundles it's own copy of `requests` so it doesn't use the one installed through `pip`.

**Comment by srshearer** [comment]
I believe you are correct. Any tips on cleaning up my path?

```
>>> import sys
>>> print sys.path
['', '/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python27.zip', '/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7', '/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-darwin', '/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac', '/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac/lib-scriptpackages', '/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-tk', '/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-old', '/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload', '/usr/local/lib/python2.7/site-packages']
```

**Comment by JonnyWong16** [comment]
Google is your friend here.

**Comment by srshearer** [comment]
Okay, I still have multiple versions of python installed which is less than ideal, but I have resolved my issue. The modules were installed into `/usr/local/lib/python2.7/site-packages` and when python would get launched from pyplex those modules were not in the path. 

I added a path insert to my import statements, so now it looks like this and is working properly…
```
import os
import sys
import argparse
import json
sys.path.insert(0, os.path.join(os.path.dirname(__file__),
                                '/usr/local/lib/python2.7/site-packages'))
import requests
```

I still don't really understand why plexpy's own included requests package isn't found, but this is working for now.
